api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -127,6 +127,26 @@
    * @since 4.0.0
    */
   default void sendMessage(final @NonNull Component message) {
+    this.sendMessage(message, MessageType.SYSTEM);
+  }
+
+  /**
+   * Sends a chat message.
+   *
+   * @param message a message
+   * @param type the type
+   * @see Component
+   * @since 4.0.0
+   */
+  default void sendMessage(final @NonNull Component message, final @NonNull MessageType type) {
+  }
+
+  /**
+   * Message types.
+   */
+  enum MessageType {
```

Comment:
Can we move this to be an independent class, outside of Audience?

Reply:
Sure.